MoneyTransferWorker/Workflow.cs

```diff
@@ -0,0 +1,68 @@
+// @@@SNIPSTART money-transfer-project-template-dotnet-workflow
+namespace Temporalio.MoneyTransferProject.Workflow;
+using Temporalio.Workflows;
+using Temporalio.Common;
+using Temporalio.MoneyTransferProject.Worker.BankingServiceExceptions;
+using Temporalio.MoneyTransferProject.Shared;
+using Temporalio.MoneyTransferProject.Activities;
+
+[Workflow]
+public class MoneyTransferWorkflow
+{
+    [WorkflowRun]
+    public async Task<string> RunAsync(PaymentDetails details)
+    {
+        // Retry policy
+        var retryPolicy = new RetryPolicy
+        {
+            InitialInterval = TimeSpan.FromSeconds(1),
+            MaximumInterval = TimeSpan.FromSeconds(100),
+            BackoffCoefficient = 2,
+            MaximumAttempts = 500,
+            NonRetryableErrorTypes = new[] { "InvalidAccountError", "InsufficientFundsError" }
+        };
+
+        string withdrawResult;
+        try
+        {
+            withdrawResult = await Workflow.ExecuteActivityAsync(
+                () => BankingActivities.WithdrawAsync(details),
+                new ActivityOptions { ScheduleToCloseTimeout = TimeSpan.FromMinutes(5), RetryPolicy = retryPolicy }
+            );
```

Comment:
```suggestion
                new ActivityOptions { StartToCloseTimeout = TimeSpan.FromMinutes(5), RetryPolicy = retryPolicy });
```

We usually recommend start to close (and don't usually want end paren on its own line)